Source/Fuse.Common/Tests/FuseTest/ObservableCollector.uno

```diff
@@ -55,7 +55,7 @@ namespace FuseTest
 			CleanSubscription();
 			if (_items == null)
 				return;
-			_subscription = _items.Subscribe(this);
+			_subscription = (ISubscription)_items.Subscribe(this);
```

Comment:
It doesn't seem like the class will actually try to do anything with that depends on ISubscription... Why is this cast useful for the test? Couldn't it just depend on the IDisposable-ism here?